projects/igniteui-angular/src/lib/grids/grid/grid.search.spec.ts

```diff
@@ -781,80 +645,72 @@ describe('IgxGrid - search API #grid', () => {
 
                 expect(highlights.length).toBe(1);
                 if (ind === 1) {
-                    expect(activeHighlight).toBe(highlights[0]);
+                    verifyActiveHighlight(0);
                 } else {
                     expect(activeHighlight).toBeNull();
                 }
                 expect((highlights[0] as HTMLElement).innerText).toEqual('12');
             });
         });
 
-        it('Search should close row edit mode', async () => {
+        it('Search should close row edit mode', () => {
             grid.primaryKey = 'ID';
             grid.rowEditable = true;
             grid.getColumnByName('Name').editable = true;
             grid.cdr.detectChanges();
             fix.detectChanges();
-            await wait();
             const row = grid.getRowByIndex(0);
             const cell = grid.getCellByColumn(0, 'Name');
 
             grid.findNext('Casey');
             grid.cdr.detectChanges();
             fix.detectChanges();
-            await wait();
-
             let highlights = cell.nativeElement.querySelectorAll('.' + fix.componentInstance.highlightClass);
             expect(highlights.length).toBe(1);
             expect(row.inEditMode).toBe(false);
             cell.nativeElement.dispatchEvent(new Event('dblclick'));
             fix.detectChanges();
-            await wait();
-
             expect(row.inEditMode).toBe(true);
 
             let cellInput = null;
             cellInput = cell.nativeElement.querySelector('[igxinput]');
             cellInput.value = 'newCellValue';
             cellInput.dispatchEvent(new Event('input'));
             fix.detectChanges();
-            await wait();
 
             grid.findNext('Casey');
             grid.cdr.detectChanges();
             fix.detectChanges();
-            await wait();
-
             highlights = cell.nativeElement.querySelectorAll('.' + fix.componentInstance.highlightClass);
             expect(highlights.length).toBe(1);
             expect(row.inEditMode).toBe(false);
         });
     });
 
-    /* ScrollableGrid */
-    describe('', () => {
-        beforeEach(async () => {
+    describe('ScrollableGrid - ', () => {
+        beforeEach(() => {
```

Comment:
Can you create this fakeAsync, because for all the components which have set width and height we need beforeEach to be fakeAsync